vissl/utils/events.py

```diff
@@ -0,0 +1,143 @@
+import json
+from collections import defaultdict
+
+from fvcore.common.file_io import PathManager
+from fvcore.common.history_buffer import HistoryBuffer
+
+
+_VISSL_EVENT_STORAGE_STACK = []
+
+
+def get_event_storage():
+    return _VISSL_EVENT_STORAGE_STACK[-1]
+
+
+def create_event_storage():
+    _VISSL_EVENT_STORAGE_STACK.append(VisslEventStorage(1))
```

Comment:
@prigoyal we put in a `1` here for starting iteration, but I think that may not work if we are resuming from a checkpoint. I think adding an optional `start_iter=0` parameter would be a good change for `create_event_storage`, but I'm not quite sure yet where I would get that from. Any pointers?

Reply:
https://github.com/MLH-Fellowship/vissl/blob/master/vissl/trainer/trainer_main.py#L174 here the `task.iteration` contains the correct start_iter (whether start from scratch or resume from a checkpoint)

Reply:
@prigoyal We are still getting an iteration value of 0 at that point. Do you think we can hop on a call any time today?

Reply:
sorry @grace-omotoso , I was on PTO but we have our scheduled call today, let's use the time then :)